tools/changelog-helper.sh

```diff
@@ -14,19 +14,18 @@ export GH_REPO=jamulussoftware/jamulus
 
 PR_LIST_LIMIT=300
 TRANSLATION_ENTRY_TEXT="GUI: Translations have been updated:"
-declare -A LANGS=(
-  [de_DE]="German"
-  [fr_FR]="French"
-  [it_IT]="Italian"
-  [nl_NL]="Dutch"
-  [pl_PL]="Polish"
-  [pt_BR]="Portuguese Brazilian"
-  [pt_PT]="Portuguese European"
-  [sk_SK]="Slovak"
-  [es_ES]="Spanish"
-  [sv_SE]="Swedish"
-  [zh_CN]="Simplified Chinese"
-)
+declare -A LANGS
+LANGS[de_DE]="German"
+LANGS[fr_FR]="French"
+LANGS[it_IT]="Italian"
+LANGS[nl_NL]="Dutch"
+LANGS[pl_PL]="Polish"
+LANGS[pt_BR]="Portuguese Brazilian"
+LANGS[pt_PT]="Portuguese European"
+LANGS[sk_SK]="Slovak"
+LANGS[es_ES]="Spanish"
+LANGS[sv_SE]="Swedish"
+LANGS[zh_CN]="Simplified Chinese"
```

Comment:
This is a manual change as `shfmt` fails to handle the previous formatting properly (each `shfmt` increases the indentation...). I don't know of a way to selectively disable `shfmt` for specific lines similar to `shellcheck` or `clang-format`. Therefore, I've chosen a different array assignment syntax, although as it is more verbose.